content/en/docs/concepts/services-networking/service.md

```diff
@@ -870,6 +879,40 @@ in those modified security groups.
 
 {{< /note >}}
 
+#### Other CLB annotations on TKE
+
+There are other annotations for managing Cloud Load Balancer on TKE as described below.
+
+```yaml
+    metadata:
+      name: my-service
+      annotations:
+        service.kubernetes.io/qcloud-loadbalancer-backends-label： `key in (value1, value2)`
```

Comment:
I'm not sure how backtiks  work in side a code block, please avoid using them if you are unsure about it.